src/App.tsx

```diff
@@ -42,6 +55,34 @@ Sentry.init({
   dsn: Config.SENTRY_DSN,
 })
 
+const segmentClient = createClient({
```

Comment:
What is the behavior if no Segment key is provided?

Reply:
[The write key is a unique identifier for each Source.](https://segment.com/docs/connections/find-writekey/).
If no Segment key is provided, then the Segment SDK can't not recognize the source and no data will be sent to the Segment cloud.

Reply:
I tested the maker app running without the segment key and no crash.